exercises/numbers/fahrenheitToCelsius/fahrenheitToCelsius.js

```diff
@@ -5,9 +5,16 @@
  * @returns {boolean} The temperature in celsius
  */
 function fahrenheitToCelsius(temp) {
-  // This is your job. :).
+  var fTemp = Farenheit 
```

Comment:
Don't just copy and hope for the best: https://www.w3resource.com/javascript-exercises/javascript-basic-exercise-11.php

The JavaScript interpreter (`node`) should be your first round of feedback.